Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -5891,7 +5891,6 @@ def extract_for_local(self):
         :rtype: ``list``
         """
         if self.mirrored_object:
-            demisto.info('Updating object {}'.format(self.mirrored_object["id"]))
```

Comment:
Do we have any other "constant" field (maybe title/name) we can use for logging? Seems useful to have this printed to logs

Reply:
No, as each object being mirrored is different, depends on the product.